src/main/java/frc/robot/RobotContainer.java

```diff
@@ -63,17 +60,43 @@ private void setDefaultCommands() {
    * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
-  private void configureButtonBindings() {}
+  private void configureButtonBindings() {
+
+    //The following code is for the primary controller
+
+
+    Sendable resetMotorEncoderCommand = new InstantCommand(() -> {
+      SwerveDriveSystem.resetAllEncoders();
+    });
+
+    Sendable printAssumedCurrentWheelAngles = new InstantCommand(() -> {
+      SwerveDriveSystem.printDriveTrainSteerMotorDegrees();
+    });
+
+    SmartDashboard.putData("Reset Motor Encoders:", resetMotorEncoderCommand);
+    SmartDashboard.putData("Current Presumed Steer Motor Angles:", printAssumedCurrentWheelAngles); //Returns positive values if the wheel turned clockwise from it's starting position. (Starting position is the wheel's front facing the front of the robot)
```

Comment:
Can we move this comment onto its own line instead of having it at the end of the line? It's too long to read.